### PR TITLE
feat(draftModel collection): add ability to exclude drafts

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: src/resources/draftModel/collection.js [hunk 1]
```diff
@@ -23,10 +23,18 @@ define(function(require) {
...
 
         _fetchLists: function(options) {
-            var req = [{ action: 'sys.storage.list' }]
+            var req = [{
+                action: 'sys.storage.list'
+            }, {
+                action: get(this, 'action')
+            }];
+
+            if (options.excludeDrafs) {
```
Comment (radiansz) [inline src/resources/draftModel/collection.js]: excludeDrafts?

File: src/resources/draftModel/collection.js [hunk 2]
```diff
@@ -23,10 +23,18 @@ define(function(require) {
         },
 
         _fetchLists: function(options) {
-            var req = [{ action: 'sys.storage.list' }]
+            var req = [{
+                action: 'sys.storage.list'
+            }, {
+                action: get(this, 'action')
```
Comment (radiansz) [inline src/resources/draftModel/collection.js]: Why get?

File: src/resources/draftModel/collection.js [hunk 3]
```diff
@@ -23,10 +23,18 @@ define(function(require) {
...
+            }];
+
+            if (options.excludeDrafs) {
+                req = req.slice(1, 2);
+            }
 
-            if (!options.excludeModels) {
-                req = req.concat({ action: get(this, 'action') });
+            if (options.excludeModels) {
+                req = req.slice(0, 1);
```
Comment (radiansz) [inline src/resources/draftModel/collection.js]: Models will stay if both excludes are true

**Comment by zouxuoz** [comment]
@radiansz fixed

**Comment by radiansz** [comment]
LGTM

**Pull request merged**